### ISSUE TITLE
Create own recipe db

### ISSUE THREAD

**Issue by Heanthor** [Heanthor/auctioneer-bot#2]
Currently reliant on a one-time recipe DB dump that came from.. somewhere? No API support outside of this one file.

Best guess how they did it: scrape wowhead via https://www.wowhead.com/profession-spells/live-only:on . Recreate this scraping algorithm so we're not tied down to one file.

**Comment by unlockedSound** [comment]
above list is going to be hard to parse out ranks from

might need this search to help pick those out: https://www.wowhead.com/recipe-items/side:3/live-only:on?filter=99;11;0

**Comment by unlockedSound** [comment]
Interesting note. Your link is to profession spells. Apparently (as best I can tell) we don’t use profession spells anymore. Nothing from legion or bfa in there. Maybe not even WoD

**Comment by unlockedSound** [comment]
using the new url gets me additional items, still far from everything

also produces a mass of items, a mass of spells, and then some other monstrosity O_o

almost to NYC so will pick this up later but the single core issue still exists:
how do we get at the full list recipes

**Comment by Heanthor** [comment]
I looked at it, I think this will work:
![image](https://user-images.githubusercontent.com/7563124/47257400-315b9c80-d45b-11e8-914d-d491eb37daae.png)

This translates into a url like: https://www.wowhead.com/profession-spells/live-only:on?filter=14:14:20;2:4:1;2000:6000:0 where 2000 and 6000 are your min and max id value. You can experiment with the range to see what makes sense, but this thing doesn't have to be extremely fast (like the item api scraper, it only has to be run once per patch when done correctly). Then, you can just up the range by a certain amount each time.

**Comment by unlockedSound** [comment]
Closed as of merge above, updates/improvements continue
